src/status_im/contexts/communities/overview/events.cljs

```diff
@@ -117,13 +117,14 @@
                      {:community community-name})}]]]})))
 
 (defn request-to-join-with-signatures
-  [_ [community-id addresses-to-reveal signatures]]
+  [_ [community-id addresses-to-reveal signatures share-future-addresses?]]
```

Comment:
Related to this comment https://github.com/status-im/status-mobile/pull/20549#issuecomment-2189357063 this event will be deleted as part of the work to remove the old join flow. I'll create the issue for us.

Reply:
Thank you!